src/test/resources/test_output.csv

```diff
@@ -0,0 +1,3 @@
+fruit;quantity
+banana;30
+apple;0
```

Comment:
```suggestion
apple;0

```

Reply:
![image](https://github.com/mate-academy/jv-fruit-shop-tests/assets/49767049/12a9aecc-e3c1-4a74-82a0-ca3465ca6e93)
don't forget to review your PR, when you see red symbol, it means there's no empty line at EOF

Reply:
not fixed